pyls/hookspecs.py

```diff
@@ -23,7 +23,12 @@ def pyls_commands(config, workspace):
 
 
 @hookspec
-def pyls_completions(config, workspace, document, position):
+def pyls_jedi_completions(config, workspace, document, position):
```

Comment:
I don't think users should be able to use both completion plugins side by side since this will probably result in duplicated completion results and other weird behaviour, or am I missing something.

I can refactor it to only use on completion hook which runs either jedi or rope depending on the config. What do you think?